freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UsersyncPoller.java

```diff
@@ -48,31 +53,28 @@
     @Inject
     private UmsEventGenerationIdsProvider umsEventGenerationIdsProvider;
 
-    @Value("${freeipa.syncoperation.poller.enabled:true}")
-    private boolean enabled;
-
     @Inject
     private EntitlementService entitlementService;
 
-    @Scheduled(fixedDelayString = "${freeipa.syncoperation.poller.fixed-delay-millis:60000}",
-            initialDelayString = "${freeipa.syncoperation.poller.initial-delay-millis:300000}")
+    @Scheduled(fixedDelayString = "${freeipa.usersync.poller.fixed-delay-millis:60000}",
```

Comment:
I am not sure whether it is intended or not, but if we have 3 pods, then this is expected to run 3 times, exactly in the same time, on the same data, doing the same thing

Reply:
This is a known issue. The operation will be rejected if another full user sync is currently running for the same environment so that the bulk of the work will not be duplicated. We have some other JIRAs for further improving the behavior.